Code/skyrim_ui/src/app/services/chat/commands.ts

```diff
@@ -0,0 +1,63 @@
+import { ChatService, MessageTypes } from "../chat.service";
+
+export interface Command {
+  readonly name: string,
+  readonly executor: (args: string[]) => Promise<void>
+}
+
+export class CommandHandler {
+
+  private Help: Command = { name: 'help', executor: async () => {
+    const cmds = [...this.commands.keys()].join(', ');
+    this.chatService.pushSystemMessage('SERVICE.COMMANDS.AVAILABLE_COMMANDS', {cmds});
+  }}
+
+  private PartyChat: Command = { name: 'party', executor: async (args) => {
+    const content = args.join(' ');
+    this.chatService.sendMessage(MessageTypes.PARTY_CHAT, content);
+  }}
+
+  private LocalChat: Command = { name: 'local', executor: async (args) => {
+    const content = args.join(' ');
+    this.chatService.sendMessage(MessageTypes.LOCAL_CHAT, content);
+  }}
```

Comment:
Move them outside the class

Reply:
moved the LocalChat and PartyChat into the ChatService, since they're both depending on that. Help command will have to stay, as it requires access to `prviate commands` and does not need to access from anywhere else.

Reply:
Didn't noticed that 🤔 well it doesn't really matter so just let it be as is